### PR TITLE
Fixes Phalcon\Http\Response getStatusCode() to return code only (int)

### PR DESCRIPTION
**Pull request by stamster**
* Type: bug fix | new feature ?
* Link to issue: #12895 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR. (not unit tests but my own code tests)

Small description of change:

Changes wrong behavior of Phalcon\Http\Response getStatusCode() method to return status code only as int instead of full string (see #12895)

Thanks

### CODE REVIEW

**Comment by Jurigag** [comment]
Update changelog please.

**Comment by stamster** [comment]
@Jurigag Where should I put it? [3.2.0] section in changelog.md, or to create new row with [3.2.1]?

**Comment by Jurigag** [comment]
There is already new row.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # [3.2.1](https://github.com/phalcon/cphalcon/releases/tag/v3.2.1) (2017-XX-XX)
 - Fixed inconsistent behaviour of `Phalcon\Config::merge` across minor version of PHP7 [#12779](https://github.com/phalcon/cphalcon/issues/12779)
 - Fixed visibility of `Phalcon\Mvc\Model\Query\Builder::{_conditionNotIn,_conditionIn,_conditionNotBetween,_conditionBetween}` to allow 3rd party libraries extend it
+- Fixed `Phalcon\Http\Response` method `getStatusCode()` to return (int) HTTP code only, instead of full string
```
Comment (Jurigag) [inline CHANGELOG.md]: Fixed `Phalcon\Http\Response::getStatusCode`

Comment (stamster) [inline CHANGELOG.md]: Yeah, I had troubles with squashing commits xD I'll open new PR.

Comment (stamster) [inline CHANGELOG.md]: #12916

**Comment by stamster** [comment]
ref: #12916

**Pull request closed without merging**